harvest/broker/__init__.py

```diff
@@ -0,0 +1,2 @@
+from harvest.broker.dummy import DummyBroker
+from harvest.broker.robinhood import RobinhoodBroker
```

Comment:
readded the init for the dummy broker

Reply:
I removed it on purpose since I noticed that it forces everyone to install all dependencies for every broker.